libraries/cyclestreets-view/src/main/java/net/cyclestreets/util/Brush.java

```diff
@@ -52,5 +52,18 @@ public static Paint createTextBrush(final int size, final int r, final int g, fi
     return paint;
   }
 
+  public static Paint createUrlBrush(final int size) {
+    final Paint paint = White;
+
+    paint.setTextAlign(Paint.Align.CENTER);
+    paint.setTypeface(Typeface.DEFAULT);
+    paint.setTextSize(size * 2);
```

Comment:
These 5 lines could be replaced with `createTextBrush(size)`

Reply:
Done, and I did a quick re-test to make sure the link appears OK in the bubble.  
I accidentally missed the file off the commit, hence the extra commit in addition to the translations - sorry!  (Let me know if you want them squashing.)

Reply:
No worries.

No need to worry about squashing your commits to the PR; we squash the whole PR into one commit as part of merging.